src/dune_lang/shell_spec.mli

```diff
@@ -0,0 +1,16 @@
+open Dune_sexp
+
```

Comment:
Add a comment here that this is only about string_with_vars and no expansion of `exec_file_shell` is performed.

Reply:
addressed in 179f7daa78c82f8665d0dbd71c0481af437f000b